### DISCUSSION TITLE
Configuation is ignored

### DISCUSSION THREAD

**Discussion by glottis9** [python-lsp/python-lsp-server#184] [Configuration]
I can't get pylsp to honor my config. I put [flake8] and/or [pycodestyle] sections in setup.cfg but still get errors in my buffer. I also have the following elisp:

(setq lsp-pylsp-configuration-sources ["flake8"]
lsp-pylsp-plugins-flake8-ignore ["D100" "D103" "E501"])

and that doesn't work either. Any idea what I need to do?

**Comment by doolio** [comment]
This sounds like an issue with your Emacs configuration. Have you raised it with the `lsp-mode` team?
